extension/src/popup/constants/history.ts

```diff
@@ -0,0 +1,3 @@
+import { createHashHistory } from "history";
+
+export const history = createHashHistory();
```

Comment:
Had to peel a couple of functions out into new files for mocking purposes. This call had been in `App.tsx` so it was pulling in a ton of other stuff I didn't want to have to mock